pareto/examples/desalination_jupyter_notebooks/mee_mvr_jupyter_notebook.ipynb

```diff
@@ -0,0 +1,243 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e89e1b65",
+   "metadata": {},
+   "source": [
+    "# MEE-MVR desalination\n",
+    "\n",
+    "A multistage evaporator with mechanical vapor recompression has been modeled in Pyomo. This notebook demonstrates how to build the model, specify inlet conditions and obtain CAPEX and OPEX post optimization.\n",
+    "The MEE-MVR model is taken from Onishi's 2017 paper on multi-effect evaporators [1].\n",
+    "\n",
+    "<br>\n",
+    "We will begin with the relevant imports:\n",
+    "<br>\n",
+    "1) Pyomo for developing the model \n",
+    "<br>\n",
+    "2) degrees_of_freedom package from IDAES core to analyse the degrees of freedom\n",
+    "<br>\n",
+    "3) 'make_mee_mvr model' from PARETO\n",
+    "<br>\n",
+    "<br>\n",
+    "[1] V. C. Onishi, et.al. “Shale gas flowback water desalination: Single vs multiple effect evaporation with vapor recompression cycle and thermal integration,” Desalination, vol. 404, pp. 230–248, Feb. 2017"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2469dd8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyomo.environ as pyo\n",
+    "from idaes.core.util.model_statistics import degrees_of_freedom\n",
+    "from pareto.models_extra.desalination_models.mee_mvr import make_mee_mvr_model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "47965656",
+   "metadata": {},
+   "source": [
+    "The inputs to the MEE-MVR process are:\n",
+    "<br>\n",
+    "1) N_evap - An integer indicating the number of evaporator stages (default = 1)\n",
+    "<br>\n",
+    "2) inputs_variables - A boolean indicating if the feed and salinity to the desalination unit are variables (default = False)\n",
+    "<br>\n",
+    "In this notebook, we will always assume our inputs are parameters ('inputs_variables' = False). "
```

Comment:
```suggestion
    "In this notebook, we will always assume our inputs are parameters (`inputs_variables` = False). "
```
Use backticks instead of single quotes to get the text to render as monospaced font in the Jupyter notebook.